src/Codeception/Module/Yii2.php

```diff
@@ -55,6 +55,8 @@ public function _before(\Codeception\TestCase $test)
     {
         $this->client = new \Codeception\Lib\Connector\Yii2();
         $this->client->configFile = \Codeception\Configuration::projectDir().$this->config['configFile'];
+        $this->client->setServerParameter('HTTP_HOST', parse_url(\Codeception\Configuration::config()['config']['test_entry_url'], PHP_URL_HOST));
```

Comment:
For example from here:
https://github.com/yiisoft/yii2-app-advanced/blob/master/tests/codeception/frontend/codeception.yml#L17